Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -387,9 +387,76 @@ namespace xr
             renderResource.DepthSwapchains.clear();
         }
 
+        Anchor CreateAnchor(Pose pose, NativeTrackablePtr trackable, XrTime time)
+        {
+            auto& apiExtensions = HmdImpl.Extensions;
+
+            if (!apiExtensions->SpatialAnchorSupported)
+                throw std::runtime_error("Spatial anchors are not supported for this device.");
+
```

Comment:
nit: for consistency please put this and the below throw in their own brackets compared to not having a bracket